app/code/Magento/Catalog/Block/Product/View/Gallery.php

```diff
@@ -233,4 +233,19 @@ private function getGalleryImagesConfig()
 
         return $this->getData('gallery_images_config');
     }
+
+    /**
+     * @param string $image
+     * @return string
+     */
+    public function getMainProductImage()
+    {
+        $image = $this->getGalleryImages()->toArray()['items'][0]['medium_image_url'];
```

Comment:
@miguelbalparda This code need some love before merging. I'm not sure how to access this data in the way that the integration tests will be happy.

Reply:
Tyvm for the heads up @cpartica!!